exampleSite/content/jobs/healers/_index.md

```diff
@@ -3,6 +3,7 @@ menu:
     main:
         name: 'Healers'   
         parent: 'jobs'
+        role: 'healer'
```

Comment:
This is for menu items in the headers/footers; you'll probably want to make this a not-nested element.

Also, could you update `exampleSite/static/admin/config.yml` to reflect this change? You can can use the [list widget](https://www.netlifycms.org/docs/widgets/#list) for this.

We use the stuff from here to inform the changes that need to go into the production editor in the `balance-static` repo.

Reply:
That's a great idea! Do you mean I should add a complete new collection? 🤔 There currently is no entry for the "parent roles", only for scholar.

Reply:
In this case, I would create a new section (much like how the scholar "section" is laid out) with some "file(s)" for different roles (in this case, we only want a healer file. Create some default object settings (for the menu items, since we don't want people in the editor to change those!), and then add a role widget.

If you feel like that's too much for this PR (because I do try to add appropriate editor changes as the theme expands in functionality), I can PR it tomorrow for you to get a feel for what I'm thinking about with it.

Reply:
Yeah I would feel more comfortable for with change if you could do that. Once I understand it I can apply future changes by my own.

Reply:
Sure thing; in the meantime, change the frontmatter to:

```yml
---
menu:
    main:
        name: 'Healers'   
        parent: 'jobs'
role: 'healer'
---
```